src/Ui/Table/Command/GetTableEntries.php

```diff
@@ -56,8 +56,9 @@ public function handle()
         /*
          * Resolve the model out of the container.
          */
-        $repository = $this->builder->getRepository();
-        
+        if (is_string(($repository = $this->builder->getRepository()))) {
```

Comment:
Have you checked it?

Reply:
```php
$repository = $this->builder->getRepository();

if (is_string($repository)) {
    $repository = app($repository);
}
```

More readale

Reply:
Yup, works.

If `$repository` is not provided in `TableBuilder` then `EntryTableRepository` is used. Otherwise `$repository` is resolved out of the container correctly and can be used (if implements `TableRepositoryInterface` -> docs should be updated as well).

Reply:
Would you mind updating docs real quick?